cmd/keycloakb/keycloak_bridge.go

```diff
@@ -836,6 +863,63 @@ func config(logger log.Logger) *viper.Viper {
 	return v
 }
 
+func checkMigrationVersion(logger log.Logger, dbParams dbConfig, migrationVersionDBConn CloudtrustDB, minMigrationVersion string) error {
+	var err error
+	migrationVersionDBConn, err = dbParams.openDatabase()
+	if err != nil {
+		logger.Log("msg", "could not create DB connection in order to check the migration version", "DB", dbParams.Database, "error", err)
+		return err
+	}
+	var flywayVersion string
+	row := migrationVersionDBConn.QueryRow(`SELECT version FROM flyway_schema_history ORDER BY installed_rank DESC LIMIT 1;`)
+	err = row.Scan(&flywayVersion)
+	if err != nil {
+		logger.Log("msg", "could not query the flyway_schema_history", "error", err)
+		return err
+	}
+
+	// compare the two versions of the type x.y (major.minor)
+	minMajorRequired := strings.Split(minMigrationVersion, ".")
+	minMajorFlyway := strings.Split(flywayVersion, ".")
+
+	majorRequired, err := strconv.Atoi(minMajorRequired[0])
+	if err != nil {
+		logger.Log("msg", "could not convert string to int", "error", err)
+		return err
+	}
+	minorRequired, err := strconv.Atoi(minMajorRequired[1])
+	if err != nil {
+		logger.Log("msg", "could not convert string to int", "error", err)
+		return err
+	}
+
+	majorFlyway, err := strconv.Atoi(minMajorFlyway[0])
+	if err != nil {
+		logger.Log("msg", "could not convert string to int", "error", err)
+		return err
+	}
+	minorFlyway, err := strconv.Atoi(minMajorFlyway[1])
+	if err != nil {
+		logger.Log("msg", "could not convert string to int", "error", err)
+		return err
+	}
+
+	// it is required for the last script version of the flyway is to "bigger" than the required version
```

Comment:
rephrase: is to "bigger" => to be "bigger ?

Reply:
Changed